src/tools/plan-generator.ts

```diff
@@ -50,23 +47,41 @@ export class PlanGenerator {
   async createPlans(): Promise<string[]> {
     const plans: string[] = [];
     for (const recipe of this.resolvedRecipes) {
-      const particles: string[] = [];
-      for (const particle of recipe.particles) {
-        particles.push(await this.createParticle(particle));
-      }
-
-      const planArgs = [ktUtils.listOf(particles)];
-      if (recipe.annotations.length > 0) {
-        planArgs.push(PlanGenerator.createAnnotations(recipe.annotations));
-      }
-
-      const start = `object ${recipe.name}Plan : `;
-      const plan = `${start}${ktUtils.applyFun('Plan', planArgs, {startIndent: start.length})}`;
-      plans.push(plan);
+      const plan = `${ktUtils.applyFun(`val ${recipe.name}Plan = Plan`, [
```

Comment:
This seems like a misuse of `applyFun`. 

A better fit would be: 
```
`val ${recipe.name}Plan = ` + ktUtils.applyFun('Plan', [...]);
```

Something like this...

Reply:
Reverted to that, as that's roughly what was here before.